### DISCUSSION TITLE
OneRowProfile is not working properly

### DISCUSSION THREAD

**Discussion by shunnl** [pazusoba/core#41] [Q&A] [answered]
When I used OneRowProfile, only 5 drops were connected.
How can fix this?

**Comment by HenryQuan** [comment]
I also found out that it only does 5 instead of the entire row in 0.7.5. This is a bug introduced by my changes to ComboProfile. Maybe consider using an older version like 0.7.2 (not sure if it works before). 😅  Sorry for the broken release as I was focusing on automation. 
I am working on a major refactor at the moment. It will take a few months but should fix all profile related issues.

**Comment by shunnl** [comment, marked as answer]
Thank you. 
I will try the older version.
